src/index.js

```diff
@@ -0,0 +1,52 @@
+import { from } from 'form-data'; // eslint-disable-line
+import './style.css'; // eslint-disable-line
+import { ToDoList } from './arr-crud'; // eslint-disable-line
+import { Task } from './arr-crud'; // eslint-disable-line
+import { dragndrop, renderTasks } from './drag-and-drop.js'; // eslint-disable-line
+
+export const toDoListArr = new ToDoList();
+
+const addToDo = document.querySelector('.add-item');
+const clearAllCompleted = document.querySelector('.btn-clear-all');
+
+function setStorage(ToDoArr) {
+  localStorage.setItem('ToDo', JSON.stringify(ToDoArr));
+}
+
+function getStorage() {
+  const ToDoArr = JSON.parse(localStorage.getItem('ToDo'));
+  const localArray = ToDoArr.map((item) => new Task(item.description, item.completed, item.index));
+  toDoListArr.setNewArr(localArray);
+  toDoListArr.arrayChanged();
+}
```

Comment:
Get function without a return value. Kindly suggest you change the name as this function is not [getting](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get).